scene/2d/sprite_2d.cpp

```diff
@@ -489,6 +535,7 @@ void Sprite2D::_bind_methods() {
 	ADD_GROUP("Animation", "");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "hframes", PROPERTY_HINT_RANGE, "1,16384,1"), "set_hframes", "get_hframes");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "vframes", PROPERTY_HINT_RANGE, "1,16384,1"), "set_vframes", "get_vframes");
+	ADD_PROPERTY(PropertyInfo(Variant::VECTOR2I, "separation", PROPERTY_HINT_NONE, "suffix:px"), "set_separation", "get_separation");
```

Comment:
```suggestion
	ADD_PROPERTY(PropertyInfo(Variant::VECTOR2I, "separation", PROPERTY_HINT_RANGE, "0,100,or_greater,suffix:px"), "set_separation", "get_separation");
```
Or some similar range value, this is supported and good to use, you still need to keep the clamping in the method though as this just limits the editor input

Reply:
I tried this but didn't see any real change in the drag behavior and ended up removing it. It still seems to drag to an obscene range with almost no mouse movement.

Readded, in any case!

Reply:
Try `"0,100,1,or_greater,suffix:px"`

Reply:
Done!